_posts/2024-05-13-social-reader-is-out.md

```diff
@@ -0,0 +1,58 @@
+title: 'Social Reader is out!'
+author: Hypha & Sutty
+date: 2024-05-13
+acknowledgement: 
+excerpt: "The Social Reader is a privacy-first personal reader for subscribing to content published on federated social media. It's ideal whether it's your first step into the Fediverse, and you want to browse without creating an account, or you just prefer to use tools that are private-by-design."
+---
+
+_The Social Reader is a privacy-first personal reader for subscribing to content published on federated social media. It's ideal whether it's your first step into the Fediverse, and you want to browse without creating an account, or you just prefer to use tools that are private-by-design._
+
+Today we are releasing **an exciting enhancement** for [Distributed Press](https://distributed.press): the **Social Reader** - a new way to read posts on the Fediverse.
+
+## The Social Reader is a privacy-first personal reader for subscribing to content published on federated social media.
+
+The Social Reader is ideal whether it's your first step into the Fediverse, and you want to browse without creating an account, or you just prefer to use tools that are private-by-design.
+
+### Skip the need to manage a digital persona with this free, no sign-up tool: just follow and access all the beautiful content you want.
+
+- **Local-first**
+- **Works offline**
+- **No account**
+
+
+
+<figure>
+    <div class='flex items-center justify-center' style="width: 100%;">
+        <img style="width:40%;" class="mt2 pt2" src="{{ '/assets/images/posts/2024-05-13-a676966.gif' | relative_url }}" alt="Social Reader Interface"/>
+    </div>
+</figure>
+
+
+
+This tool also **works without internet connectivity:** you'll have your own database locally, so you don't need to go online to read posts.
+
+In addition, it has native support for **subscribing via peer-to-peer feeds** if you use [Agregore](https://agregore.mauve.moe/), [Galacteek](https://galacteek.gitlab.io/es) or any other **p2p browser**. This makes use of the peer-to-peer published versions of people's sites, fostering the ecosystem to gradually adopt decentralized publishing flows. Consequently, streams are more resistant to outages or attacks, given that their data is distributed across multiple nodes. 💪
+
+
+### Try it out here: [reader.distributed.press](https://reader.distributed.press)
+
+Watch a brief demo 👀
+
+
+<video controls style="width:100%">
+    <source src="{{ 'assets/images/posts/2024-05-13-social_reader.mp4' | relative_url }}" >
+</video>
+
+Curious about operational details? Check out our [documentation](https://docs.distributed.press/social-reader)
+
+#### _Disclaimer: it is important to note that Distributed Press does not host any of the content accessed through Social Reader. Therefore, Distributed Press bears no legal responsibility for the appropriateness, accuracy, or legality of the content accessed or distributed through the use of this tool._
```

Comment:
We should double check this with a lawyer or similar at some point. Also might mean we need a privacy policy? Can add it to the backlog tho.